packages/compass-signtool/src/sign.ts

```diff
@@ -0,0 +1,49 @@
+import path from 'path';
+import { exec } from 'child_process';
+import { promisify } from 'util';
+import { SSHClient } from './ssh-client';
+import { SigningClient } from './signing-client';
+import { assertRequiredVars } from './utils';
+
+const execAsync = promisify(exec);
+
+async function getSpawnnedHostName() {
+  // todo: this will fail
+  // return 'ec2-3-254-76-71.eu-west-1.compute.amazonaws.com';
+  const command = `tr -d '"[]{}' < ${path.resolve(
+    process.cwd(),
+    './spawned_hosts.yml'
+  )} | cut -d , -f 1 | awk -F : '{print $2}'`;
+  const { stdout } = await execAsync(command);
+  return stdout.trim();
+}
+
+const getSshClient = async () => {
+  const hostname = await getSpawnnedHostName();
+  const sshClient = new SSHClient({
+    host: hostname,
+    port: 22,
+    username: 'ubuntu',
+    // todo: this will fail?
+    privateKey: path.resolve('/.ssh/mcipacker.pem'),
```

Comment:
A thought: later we probably want to make this one, and some of the other stuff that are directly depending on how the ssh host is configured, or how our CI is setup to be arguments passed to this library.

The reasoning here is that if this package is going to be shared and live outside of the compass and mongosh repos, if we have to change the signing host, or paths change in evergreen, we don't need to also change the package at all.